### PR TITLE
[DOCS-142] SAML SSO for All Tiers

### PR DESCRIPTION
**Pull request by ana-dashuk-cobalt**
@molfinn Please review the updates describing changes to SAML SSO.

This is how the changes will appear on the portal: https://deploy-preview-140--cobalt-docs.netlify.app/getting-started/sign-in/

`Learn more about [configuring SAML SSO](https://cobaltio.zendesk.com/hc/en-us/sections/360012774052--SAML-SSO-).` The link here will be changed as soon as we finalize [DOCS-157](https://zombie.atlassian.net/browse/DOCS-157).

### CODE REVIEW

**Review by molfinn — APPROVED** [review]

File: content/en/Getting started/glossary.md [hunk 1]
```diff
@@ -121,9 +121,7 @@ An _Organization Owner_ is the administrator for a customer organization within
 
 - An _Organization Owner_ may also be a [_Pentest Team Member_](#pentest-team-member).
 
-- If allowed by their {{% ptaas-tier %}}, they can also manage
-[multi-factor authentication (MFA)](#multi-factor-authentication) as well as
-[SAML](#security-assertion-markup-language) settings for the users in their organization.
+- Manage [multi-factor authentication (MFA)](#multi-factor-authentication) as well as [SAML](#security-assertion-markup-language) settings for the users in their organization.
```
Comment (mjang-cobalt) [inline content/en/Getting started/glossary.md:124]: And now I realize that I was not consistent here.... What do you think? We could:

- Change this to 2FA
- Explain that in our app, users can set up MFA through "Two-factor authentication"

**Review by mjang-cobalt — APPROVED** [review]
Approving with a comment/question. @ana-dashuk-cobalt , I'll trust your decision.

File: content/en/Getting started/glossary.md (hunk 1 above)
Comment (ana-dashuk-cobalt) [reply content/en/Getting started/glossary.md:124]: @mjang I would add one sentence to Glossary > Multi-factor Authentication:

We support two-factor authentication with codes from a mobile authenticator app.

Comment (mjang-cobalt) [reply content/en/Getting started/glossary.md:124]: Sounds good to me. I might even make it shorter:

```suggestion
- Manage [multi-factor authentication (MFA)](#multi-factor-authentication) as well as [SAML](#security-assertion-markup-language) settings for the users in their organization. We support two-factor authentication with a mobile authenticator app."
```

**Pull request merged**